agent/custom/bbc_action.py

```
@@ -608,24 +612,28 @@ def _handle_popups(self, messages: list, support_order_mismatch: bool,
+        3. 心跳成功后截图并运行"助战错误识别"pipeline，识别到助战错误则停止BBC脚本并正常结束任务
+        4. 弹窗处理由回调函数在后台线程完成，通过 state['finished'] 通知主线程
+        5. 当 state['finished'] 为 True 时，返回弹窗信息
         """
-        
+
         # 获取 manager 实例
         manager = get_manager()
-        
-        # 主线程：只做心跳检查
+
+        # 主线程：心跳检查 + 截图识别
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)